pom.xml

```diff
@@ -168,7 +168,7 @@
         <protobuf.plugin.version>0.6.1</protobuf.plugin.version>
         <os.plugin.version>1.6.2</os.plugin.version>
         <junit.version>4.13.1</junit.version>
-        <mockito.version>1.10.19</mockito.version>
+        <mockito-core.version>2.28.2</mockito-core.version>
```

Comment:
nit:
AFAIK the mockito libraries have a single common version, we could keep using mockito.version as the property name, even if we only pull mockito-core.